notebooks/CubevizExample.ipynb

```diff
@@ -27,7 +27,7 @@
     "# This file is originally from https://data.sdss.org/sas/dr14/manga/spectro/redux/v2_1_2/7495/stack/manga-7495-12704-LOGCUBE.fits.gz\n",
     "# but has been modified to correct some inconsistencies in the way units are parsed\n",
     "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
-    "viz.load_data(fn)"
+    "viz.load_data(fn)\n"
```

Comment:
FYI -- I am going to drop the changes here in my rebase. So the cleaned up PR won't show anymore changes to the example notebook, just `parsers.py`.